### PR TITLE
Feature: Added Navigation Component

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added Navigation bar.
- Added text, search bar, buttons, icons.

Live Link: https://modest-wright-07d9a0.netlify.app/

### CODE REVIEW

File: components/navigation/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            <p class="head">MyWebsite</p>
+            <input class="search" type="text" placeholder="Search"></input>
+
+            <div class="nav-right-container">
+                <button class="btn">Login</button>
+                <i class="far fa-heart fa-lg"></i>
+                <i class="fas fa-shopping-cart fa-lg"></i>
+            </div>
+        </nav>
+
```
Comment (manujagobind) [inline components/navigation/index.html:27]: Remove extra line breaks.

**Review by manujagobind — APPROVED** [review]

**Review by sahiba0915 — COMMENTED** [review]
looks good to me

File: components/navigation/index.html (hunk 1 above)
Comment (poojakelkar) [reply components/navigation/index.html:27]: sure, Thanks

**Comment by poojakelkar** [comment]
Thanks @sahiba0915

**Pull request merged**